truenas/webui
# NAS-112914 / 22.02-RC.2 / NAS-112914: Removing Theme editor and other code cleanup

Changes: 
- Removes theme editor (was hidden for most users) and moves some code around.
- Updates api calls used to delete devices.

Testing:
- Check that Preferences from top right menu work correctly.
- Check that removing devices from a virtual machine in Virtualization works correctly.
